assets/custom.js

```diff
@@ -1,22 +1,45 @@
-//Shows dropdown box for 'Availability' for desktop
-$(".filter__container").click(function(){
-    const filter__activeContainer = $(this).children().last();
-    filter__activeContainer.toggleClass("display-block");
+//Shows Dropdown box for 'Availability' for DESKTOP
+$(".filter__label").click(function(){
+    const filterActiveContainer = $(this).parent().find('.filter__activeContainer');
+    filterActiveContainer.toggleClass("display-block");
 });
 
-//Shows Menu for mobile when 'Filter and Sort' is clicked
+//Shows Menu for 'Filter and Sort' for MOBILE
 $(".filterAndSortBy__label").click(function(){
     $(this).next().find(".activeContainer").toggleClass("display-block");
     $('.filterAndSortBy__backgroundCover').toggleClass("display-block");
 });
 
-// Hides Menu for mobile when 'Back' is clicked
+// Hides Menu when 'Back' is clicked for MOBILE
 $(".activeContainer__backContainer").click(function(){
     $(".activeContainer").toggleClass("display-block");
     $('.filterAndSortBy__backgroundCover').toggleClass("display-block");
 })
 
-// Shows 'availablity' drop down for mobile
+// Hides Menu container when user clicks outside of container for DESKTOP and MOBILE
+$(document).mouseup(function(e){
```

Comment:
just curious about using `mouseup` and `click` - is there a difference?

Reply:
In this instance; I don't think it would matter. I just copied this from a stack overflow question.